core/dbt/parser/base.py

```diff
@@ -310,10 +310,11 @@ def update_parsed_node_config(
             else:
                 parsed_node.docs = Docs(show=docs_show)
 
-        # If we have "contract" in the config, copy to node level, for backwards
+        # If we have contract in the config, copy to node level, for backwards
         # compatibility with earlier node-only config.
-        if config_dict.get("contract", False):
-            parsed_node.contract = True
+        if "contract" in config_dict and config_dict["contract"]:
+            # breakpoint()
```

Comment:
nit: this was probably left in by accident!

Reply:
oops.  fixed.